sdk/core/azure_core_test/src/tracing.rs

```diff
@@ -225,17 +245,37 @@ pub fn check_instrumentation_result(
     }
 }
 
+/// Information about an expected span. Used to assert span properties.
+#[derive(Debug)]
```

Comment:
Should this be SafeDebug? 

ditto for other structs in the file

Reply:
Only needed for customer-facing APIs. This crate doesn't even get published, and we want the full fidelity of Debug for stuff like that.